src/components/tooltip/tooltip.spec.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import PropTypes from 'prop-types';
+import { act } from 'react-dom/test-utils';
```

Comment:
I've spent so much time battling this over the last week...

Reply:
Hmm interesting...

Reply:
this thread is helpful

https://github.com/facebook/react/issues/15379

TL:DR they have added an async act in the react 16.9 which should drop shortly

Reply:
Yeah but RTL already uses `act`. But I guess updating the timers lies "outside" of RTL scope...

Reply:
the worst is combining async actions + timers. It's really very hard to test with RTL